tests/helper_test.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import unittest
+
+from chain.params import ROUND_DURATION
+from crypto.private import Private
+from node.behaviour import Behaviour
+from node.block_signers import BlockSigner
+from node.node import Node
+from tools.time import Time
+
+
+class HelperTest(unittest.TestCase):
```

Comment:
This should be named "TestHelper", because it is not testing helper. It helps to test.

Reply:
Agree